docs/docs/100-reference/01-command-line/acorn_run.md

```diff
@@ -5,47 +5,57 @@ title: "acorn run"
 
 Run an app from an image or Acornfile
 
-```
-acorn run [flags] IMAGE|DIRECTORY [acorn args]
-```
+### Synopsis
```

Comment:
My goal was to keep the help vs advanced help strings within the cmd struct but this may cause it to not be possible. are we okay with moving it to more of a custom string within the setHelpFunc and maintain the advanced help string through there? @cjellick